OpenRA.Game/Network/OrderIO.cs

```diff
@@ -65,6 +65,14 @@ public static void Write(this BinaryWriter w, CPos cell)
 		{
 			w.Write(cell.X);
 			w.Write(cell.Y);
+			w.Write(cell.Layer);
+		}
```

Comment:
Writing all three components of CPos here.

Reply:
I guess this all predates the 2.5D world.

Reply:
Weren't layers only added with subterranean/tunnel movement?

Reply:
https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Game/CPos.cs#L24

Reply:
Then yes, they were. :D